### DISCUSSION TITLE
ModuleNotFoundError: Module not found: Error: Can't resolve 'next/Image'

### DISCUSSION THREAD

**Discussion by yoryi** [vercel/next.js#21266] [Help]
I am getting error in deploying very simple Next.js project that the next/image is used to the Digital Ocean.
Here is screenshot of it. 
![image](https://user-images.githubusercontent.com/16271026/104871265-980ce080-5918-11eb-800d-5561906bf491.png)

I appreciate of Next.js community help.
Thanks.

**Comment by Sandstedt** [comment]
What version of next do you use @yoryi? Then next/image component is quite new.

**Comment by JOEMN96** [comment]
Anyone facing this issue ?

**Comment by JOEMN96** [comment]
![image](https://user-images.githubusercontent.com/56827756/129236748-e4c57ce4-e264-43f9-8e4f-6ed43f942dfb.png)
Same Error Here !

**Reply by Sandstedt** [reply]
Oh. it's not next/Image, it's next/image, with a small I. Will work on Mac OS, but not on windows and linux due to case sensitive filesystems. @JOEMN96 @yoryi

**Reply by JOEMN96** [reply]
Geez ! Thanks my brother !
